ui/panels-plugin/src/plugins/bar-chart/utils.test.ts

```diff
@@ -0,0 +1,104 @@
+// Copyright 2023 The Perses Authors
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { BarChartData } from '@perses-dev/components';
+import { calculatePercentages, sortSeriesData } from './utils';
+
+const MOCK_DATA: BarChartData[] = [
+  {
+    label: 'Label 1',
+    value: 30,
+  },
+  {
+    label: 'Label 2',
+    value: 20,
+  },
+  {
+    label: 'Label 3',
+    value: 0,
+  },
+  {
+    label: 'Label 4',
+    value: 50,
+  },
+];
+
+describe('calculatePercentages', () => {
```

Comment:
Yay thanks for adding tests. Maybe also good to add test cases with null values?

Reply:
Good idea, updated the tests dfd71b6603b5cca95f0bbe8dbcfa137375d4f47d